### PR TITLE
[MRG+2] Minor tab colour fix

### PR DESCRIPTION
**Pull request by QuLogic**
Just a minor correction to the prefix in one last location. Also remove a redundant `OrderedDict`.

### CODE REVIEW

File: lib/matplotlib/colors.py [hunk 1]
```diff
@@ -260,7 +260,7 @@ def to_hex(c, keep_alpha=False):
 ### Backwards-compatible color-conversion API
 
 cnames = CSS4_COLORS
-COLOR_NAMES = {'xkcd': XKCD_COLORS, 'css4': CSS4_COLORS, 'tc': TABLEAU_COLORS}
+COLOR_NAMES = {'xkcd': XKCD_COLORS, 'css4': CSS4_COLORS, 'tab': TABLEAU_COLORS}
```
Comment (tacaswell) [inline lib/matplotlib/colors.py]: This isn't used internally anymore and is not in any non-rc release.  Do we just want to remove it?

Comment (QuLogic) [inline lib/matplotlib/colors.py]: I think you're right; what was this for, anyway?

Comment (QuLogic) [inline lib/matplotlib/colors.py]: Looks like it was removed in #6382, but left for backwards-compatibility. Should we just remove `tab` here?

Comment (tacaswell) [inline lib/matplotlib/colors.py]: but it went in from https://github.com/matplotlib/matplotlib/pull/5775

Comment (anntzer) [inline lib/matplotlib/colors.py]: +1 for removing it if we can (but you probably could guess my vote).

Comment (QuLogic) [inline lib/matplotlib/colors.py]: Ah, I see, I didn't look back far enough.

**Comment by tacaswell** [comment]
@QuLogic I took the liberty of just removing it.

**Pull request merged**